src/__tests__/index.ts

```diff
@@ -120,23 +136,25 @@ describe('createVerifiableCredential', () => {
   it('calls functions to validate optional fields if they are present', async () => {
     expect.assertions(1)
     const timestamp = Math.floor(new Date().getTime())
-    await createVerifiableCredentialJwt({ ...verifiableCredentialPayload, exp: timestamp }, did)
+    await createVerifiableCredentialJwt({ ...verifiableCredentialPayload, exp: timestamp }, issuer)
     expect(mockValidateTimestamp).toHaveBeenCalledWith(timestamp)
   })
 })
 
 describe('createPresentation', () => {
+  const holder = ethrDidIssuer
```

Comment:
hm, ok I feel I understand the reasoning a little more. I feel we are still hiding some of the ambiguity of our "Issuer Type" here. A Holder is a "Issuer" of a Presentation. But a Holder is generally not the issuer of a VC. I think we should make clear in Veramo what the difference between these two Issuer Definitions is (but not in this PR).